src/main/java/stsjorbsmod/powers/FragilePower.java

```diff
@@ -16,23 +19,41 @@ public FragilePower(final AbstractCreature owner, final int turnsUntilSnap) {
 
         this.owner = owner;
         this.amount = turnsUntilSnap;
+        this.amount2 = MemoryManager.forPlayer(owner).countCurrentClarities();
```

Comment:
Seeing the "1" there as the number of clarities without any tooltip text on the power explaining what the number was or how it related to the numbers in the tooltip felt confusing to me; I think it would be better to not use an amount2 at all

Reply:
Sure, I'll use a different variable.